src/main/java/ro/msg/learning/shop/controller/RevenueController.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import ro.msg.learning.shop.dto.ReveneuDTO;
+import ro.msg.learning.shop.services.ReveneuServices;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class RevenueController {
+    private final ReveneuServices reveneuServices;
+
+    @RequestMapping(value = "/reveneu", method = RequestMethod.GET)
+    public List<ReveneuDTO> getAllProducts() {
```

Comment:
Method name does not match the content :) I guess you copy-pasted it from the Product Controller :P